doc/source/whatsnew/v0.20.0.txt

```diff
@@ -53,6 +53,9 @@ Other API Changes
 Deprecations
 ^^^^^^^^^^^^
 
+- ``Series.repeat()`` has deprecated the ``reps`` parameter in favor of ``repeats`` (:issue:`12662`)
+- ``Index.repeat()`` and ``MultiIndex.repeat()`` have deprecated the ``n`` parameter in favor of ``repeats`` (:issue:`12662`)
+- ``Categorical.searchsorted()`` and ``Series.searchsorted()`` have deprecated the ``v`` parameter in favor of ``key`` (:issue:`12662`)
```

Comment:
so we typically will use ``value`` in this case (not sure why it was ``key`` in the timedelta one). We are searching the ``.values`` of the object, and NOT the index (where we typically use ``key``).